sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -458,10 +458,27 @@ def infer_return_type_func(f, input_types, debug=False, depth=0):
           pop_count = arg + 2
           return_type = Any
         elif opname == 'CALL_FUNCTION_EX':
-          # TODO(udim): Handle variable argument lists. Requires handling kwargs
-          #   first.
-          pop_count = (arg & 1) + 3
```

Comment:
In new version we increment pop_count by has_kwargs + 2, do you think old version just had a typo?  
```
          pop_count = (arg & 1) + 3	
```

Reply:
Not a typo but a bug. In python `stack[-pop_count:]` doesn't trigger an error even though `stack[-pop_count]` is out of bounds.